### ISSUE TITLE
More Datasticks for Seismic Prospecting

### ISSUE THREAD

**Issue by richardhendricks** [GTNewHorizons/NewHorizons#2223]
#### Which modpack version are you using?
2.0.1.3
#
#### What did you try to do, and what did you expect to happen?
Searching for Mica ore. With new ore changes, ores in general are more plentiful, but searching is still painful. 
#
#### What do you suggest instead/what changes do you propose?
Currently, the next level beyond "dig to bedrock" is the seismic prospector.  However, that requires a datastick, explosives, and machines to process.  I am OK with the explosives and post-processing requirements.  These add automation requirements, which are mostly fine. The problem is that the datasticks are too high tier IMO.

The seismic prospector quest gives you one, which isn't enough to really streamline ore searching. 

My preferences:

1. Remove selection from quest rewards, always grant datasticks.

2. Increase number of datasticks rewarded.  I think 4 is enough to make it worthwhile to search with. This can be done either directly in the Seismic Prospector quests, or indirectly in some of the orevein search quests. I think 2 granted for the quest, and 1 per orevein search quest is fair. Or just reward 4 in the quest.

3. Lower the crafting requirements for the datastick to at least HV, but maybe LV since the Seismic Prospector is craftable at LV. I want to check on detailed crafting requirements, would be nice if it could be used to search for redstone/lapis, the first two painful veins to locate. Not sure though since the sensors require certus quartz from the Nether.

4. 2&3 could be done together, or either/or.  

5. Adjust the post-scan recipes so they are more automation-friendly. Scanner and printer should be fast, assembler can be slowed down to compensate. The printer step requires manual interaction, which is unfortunate, but if the scanner/printer steps are faster, a player can return with a batch of sticks, process them in the scanner, process them in the printer, and then dump the papers in the assembler while they go to the next batch of chunks.

Advantages:
This gets players away from the grinding dig down/move over/dig down/move over cycle, and using technology/automation sooner in the gameplay. Also, it encourages exploration and setting up bases in remote areas to save on return trip time. I like the idea of players getting a horse/donkey and going on a prospecting expedition at LV, killing cows along the way for book covers, eventually getting gliders/copterpacks/etc. It will also encourage automation of explosives and paper sooner, which is a better grind IMO than orevein searching.

Disadvantages:
Removes the hours of orevein searching. Yes, they are a small part of the overall hours played, but they are smashed right at the front of the experience. 20 hours spread out over 1000 isn't bad, 20 hours in the first 100 sucks. 
Might result in more people playing, and staying around after starting the pack.

**Comment by Dream-Master** [comment]
Yes I agree will change quest rewards. Long time ago Johannes has the idea to add a disk to gt (cheaper stick alternative) but this was never released.

**Comment by richardhendricks** [comment]
Thanks.  So you will leave the recipe as-is and do 4 datasticks as the reward?

**Comment by Dream-Master** [comment]
Maybe 4 is a bit high will add 2 and another 2 for the other quest. (Ore drill)

**Comment by Dream-Master** [comment]
Yes recipe I leave like it is. maybe you could help me add a cheaper disk item with low functions then data stick? @richardhendricks

**Comment by richardhendricks** [comment]
That might be a while, I've never done something like that. 
Which functions do you want to keep/remove on the new item? What tier would you want it to target?

How about a "floppy disk" that uses polyethylene, magnetite dust, glue, and a little bit of steel?  That would make it mid-HV tier.

**Comment by richardhendricks** [comment]
Okay, will create a separate ticket for adding a new low-tier data storage device.

**Comment by Dream-Master** [comment]
Thanks @richardhendricks

**Comment by richardhendricks** [comment]
YEah, I tried doign some prospecting with the seismic prospector.  I had 7 scans that I stuck in the assembler at the same time to turn into books, they turned into 2 individual books and then 5 identical books.  I suspect the identical books probably had 0 oil and ores?  Still, shouldn't have done that. :(

**Comment by richardhendricks** [comment]
How about this idea - change the area scanned from one chunk to 9x9 chunks?  That would cover 9 oreveins at one time.  We could leave the results as per chunk, or mix them together. Digging 9 chunks horizontally would take ~20 minutes, so flying and doing this once or twice per trip is acceptable.

**Comment by Dream-Master** [comment]
Yes this is a good idea because how it is now the process is to costly. The CD low data stick have then a lower value.

**Comment by Dream-Master** [comment]
change the Quest reward now
